airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -257,15 +258,32 @@ def _get_secret(self, path_prefix, secret_id: str) -> Optional[str]:
             return response.get('SecretString')
         except self.client.exceptions.ResourceNotFoundException:
             self.log.debug(
-                "An error occurred (ResourceNotFoundException) when calling the "
-                "get_secret_value operation: "
+                f"ResourceNotFoundException: {error_msg}: "
                 "Secret %s not found.",
                 secret_id,
             )
             return None
-        except self.client.exceptions.AccessDeniedException:
+        except self.client.exceptions.InvalidParameterException:
             self.log.debug(
-                "An error occurred (AccessDeniedException) when calling the get_secret_value operation",
+                f"InvalidParameterException: {error_msg}",
+                exc_info=True,
+            )
+            return None
+        except self.client.exceptions.InvalidRequestException:
+            self.log.debug(
+                f"InvalidRequestException: {error_msg}",
```

Comment:
Can you change the the logging statements to use %-formatting instead of f-strings please? This makes sure that the string formatting is deferred until it cannot be avoided. [This is a helpful reference](https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/) which hopefully explains the practice here.

Reply:
Right!